.github/workflows/ci.yml

```diff
@@ -544,4 +544,4 @@ jobs:
         with:
           allowUpdates: true
           artifacts: "**/*"
-          token: ${{ secrets.GITHUB_TOKEN }}
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Unnecessary change (also removing EOL at EOF).

Reply:
That's what jinja outputs